site/en/tutorials/keras/classification.ipynb

```diff
@@ -637,7 +663,9 @@
         "id": "yWfgsmVXCaXG"
       },
       "source": [
-        "It turns out that the accuracy on the test dataset is a little less than the accuracy on the training dataset. This gap between training accuracy and test accuracy represents *overfitting*. Overfitting is when a machine learning model performs worse on new, previously unseen inputs than on the training data."
+        "It turns out that the accuracy on the test dataset is a little less than the accuracy on the training dataset. This gap between training accuracy and test accuracy represents *overfitting*. Overfitting is when a machine learning model performs worse on new, previously unseen inputs than on the training data. When the trained model 'memorises' the training data, it's accuracy on test data reduces as compared with training data and model is said to be overfitting the training data. For more information, see below:\n",
```

Comment:
> When the trained model 'memorises' the training data, it's accuracy on test data reduces as compared with training data and model is said to be overfitting the training data.

This sentence is a bit difficult to parse. Can you please shorten/clarify?
Also, use en-US spelling ("memoize") and "it's" = "its"